lms/templates/courseware/courses.html

```diff
@@ -45,15 +65,36 @@
   </header>
 
   <section class="courses-container">
+
+    <div id="discovery-form">
```

Comment:
It's super helpful to add an ```role="search"``` and ```aria-label="course"``` to this div. A screen reader will list it in the navigable regions and will announce it as Course Search.